pipelines/docker-build-multi-platform-oci-ta/README.md

```diff
@@ -98,6 +98,7 @@ This pipeline is pushed as a Tekton bundle to [quay.io](https://quay.io/reposito
 |docker-auth| unused| | |
 |image-digest| Image digest to scan.| None| '$(tasks.build-image-index.results.IMAGE_DIGEST)'|
 |image-url| Image URL.| None| '$(tasks.build-image-index.results.IMAGE_URL)'|
+|scan-threads| Number of threads to run in clamscan parallel. Should be <= 8.| 1| |
```

Comment:
I think that this should be located within clamavscan readme, the way it is defined at the moment is within each build type, but there is no refference for the variable directly to the clamav scan.

Reply:
You should be able to regenerate the task READMEs with https://github.com/konflux-ci/build-definitions/blob/main/hack/generate-readme.sh. This will output the README to stdout, you can redirect that to output back to the README file.

Reply:
@jsztuka Great catch! Ok I regenerated the README specifically for clamav-scan task, using the script recommended by @arewm and commited.  Can one of you click off the tests again?